server/portal/apps/accounts/views.py

```diff
@@ -91,6 +91,7 @@ def get_profile_data(request):
         'history': history,
         'licenses': manage_licenses(request),
         'integrations': manage_applications(request),
+        'setup_complete': django_user.profile.setup_complete
```

Comment:
Can we make this setupComplete for consistency?

Reply:
👍